### PR TITLE
Transifex don't return error code on error...

### PR DESCRIPTION
**Pull request by sbrunner**
See also: https://github.com/c2corg/v6_ui/commit/0d3bafcadfc3bddb315fe550ef9b1ce3f2d71590

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -232,7 +232,8 @@ COMPILE_CATALOG = node_modules/.bin/compile-catalog
 endif
 
 # i18n
-TX_VERSION = $(shell $(VENV_BIN)/python -c "print('_'.join(__import__('pkg_resources').get_distribution('c2cgeoportal').version.split('.')[0:2]))")
+#TX_VERSION = $(shell $(VENV_BIN)/python -c "print('_'.join(__import__('pkg_resources').get_distribution('c2cgeoportal').version.split('.')[0:2]))")
+TX_VERSION = master
```
Comment (gberaudo) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Is this correct?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 2]
```diff
@@ -729,6 +730,11 @@ $(TX_RC):
 	mkdir -p $(dir $@)
 	$(VENV_BIN)/tx pull -l $* --force
 	$(TOUCHBACK_TXRC)
+	test -s $@
+
+.build/locale/en/LC_MESSAGES/gmf.po:
+	$(PRERULE_CMD)
+	@echo "The en language is the source language, than the file steal be empty"
```
Comment (gberaudo) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: What do you mean?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Actuarially yes but I don't like it, I will fix it in ngeo first :-)

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Requires https://github.com/camptocamp/ngeo/pull/1881 to undo that :-)

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: than => then

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: so the file must stay empty

**Pull request merged**